soccer/planning/MotionInstant.hpp

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+#include <Geometry2d/Point.hpp>
+
+namespace  Planning {
+/**
+ * This class represents a instant on a motion. It cointains a position and velocity vector
+ */
+    struct MotionInstant {
+
+        MotionInstant() : pos(), vel() {}
```

Comment:
But the extremely slight perfomance benefit!
I don't know actually. It looks slightly less pretty with = Geometry2d::Point() everywhere. Do we have a static empty Point variable?